### DISCUSSION TITLE
Send links with timestamp

### DISCUSSION THREAD

**Discussion by Smallinger** [Zibbp/ganymede#262] [Ideas]
like the youtube feature, http://ganymede/vods/64ebadc3-320c-11ee-be29-0242ac160005?t=96

96 = 1:36

so you can send a specific timestamp to a other person.

**Comment by Zibbp** [comment]
Good idea. I'll probably add it to the video menu that's on the video card or the bottom right of the page when watching a video. If there is a time stamp in the URL it will take precedence over any playback history that user may have.

**Reply by Zibbp** [reply]
The copy to clipboard API requires HTTPS. If the user isn't connected via HTTPS and clicks the "share" button, a popup will be displayed instead for the user to copy the URL.

This has been released in [ganymede-frontend:v1.4.3](https://github.com/Zibbp/ganymede-frontend/releases/tag/v1.4.3).
